src/main/java/org/fcrepo/migration/validator/api/ValidationReportSummary.java

```diff
@@ -17,10 +17,33 @@
  */
 package org.fcrepo.migration.validator.api;
 
+import java.util.Collection;
+import java.util.HashSet;
+
 /**
  * A data class defining all report wide summary information
  *
  * @author dbernstein
+ * @author awoods
+ * @since 2020-12-17
  */
 public class ValidationReportSummary {
+
+    private Collection<String> objectReportFilenames = new HashSet<>();
```

Comment:
I'm not sure I understand the intent here but I'm not too worried about it leaving it in here.  But I am curious what your thoughts are here.

Reply:
This "collection" provides the filenames of the object-level HTML output reports which are linked into the top-level HTML report:
- https://github.com/fcrepo-exts/fcrepo-migration-validator/blob/main/src/main/java/org/fcrepo/migration/validator/report/HtmlReportHandler.java#L127-L128
and
- https://github.com/fcrepo-exts/fcrepo-migration-validator/blob/main/src/main/resources/templates/summary.ftl#L18-L20